bundles/org.openhab.ui/web/src/pages/settings/things/thing-details.vue

```diff
@@ -107,12 +107,12 @@
           </f7-col>
         </f7-block>
 
-        <f7-block class="block-narrow" v-if="ready && !error && thingType && thingType.UID.indexOf('zwave') === 0">
+        <f7-block class="block-narrow" v-if="ready && !error && thingType && (thingType.UID.indexOf('zwave') === 0 || configActions.length > 0)">
           <f7-col>
-            <f7-block-title>Z-Wave</f7-block-title>
+            <f7-block-title>Actions</f7-block-title>
             <f7-list>
-              <f7-list-button color="blue" title="View Network Map" @click="openZWaveNetworkPopup" />
-              <f7-list-button color="blue" v-for="action in zwaveActions" :key="action.name" :title="action.label" @click="doZWaveAction(action)" />
+              <f7-list-button color="blue" title="View Network Map" @click="openZWaveNetworkPopup" v-if="thingType.UID.indexOf('zwave') === 0" />
+              <f7-list-button :color="action.verify? 'yellow' : 'blue'" v-for="action in configActions" :key="action.name" :title="action.label" @click="doConfigAction(action)" :tooltip="action.description" />
```

Comment:
```suggestion
              <f7-list-button :color="action.verify ? 'yellow' : 'blue'" v-for="action in configActions" :key="action.name" :title="action.label" @click="doConfigAction(action)" :tooltip="action.description" />
```

Reply:
commited as-is (will squash later) - or feel free to squash if/when merging